src/constants.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  Handedness: Object.freeze({
+    NEUTRAL: "neutral",
```

Comment:
FYI: We landed the API-side change to "none" on Tuesday when there were no objections on the call

Reply:
Yeah I figured I'd handle that in a separate PR if that's ok